build-logic/src/main/kotlin/nessie-jacoco.gradle.kts

```diff
@@ -56,8 +56,6 @@ if (plugins.hasPlugin("io.quarkus")) {
       excludeClassLoaders =
         listOf("*QuarkusClassLoader") + (if (excluded != null) excluded else emptyList())
     }
-    systemProperty("quarkus.jacoco.report", "false")
-    systemProperty("quarkus.jacoco.reuse-data-file", "true")
```

Comment:
Was it fixed in Quarkus?

Reply:
There's nothing to be fixed - it warns about non-consumed configs

Reply:
Right, but we added this config a while ago to fix javaco reports. Do we not have the same problem now?